core/server/common/src/main/java/alluxio/web/WebInterfaceBrowseLogsServlet.java

```diff
@@ -119,48 +116,6 @@ protected void doGet(HttpServletRequest request, HttpServletResponse response)
     String requestFile = request.getParameter("path");
 
     if (requestFile == null || requestFile.isEmpty()) {
-      // List all log files in the log/ directory.
-
-      List<UIFileInfo> fileInfos = new ArrayList<>();
-      File[] logFiles = logsDir.listFiles(LOG_FILE_FILTER);
-      if (logFiles != null) {
-        for (File logFile : logFiles) {
-          String logFileName = logFile.getName();
-          fileInfos.add(new UIFileInfo(new UIFileInfo.LocalFileInfo(logFileName, logFileName,
-                  logFile.length(), UIFileInfo.LocalFileInfo.EMPTY_CREATION_TIME,
-                  logFile.lastModified(), logFile.isDirectory())));
-        }
-      }
-      Collections.sort(fileInfos, UIFileInfo.PATH_STRING_COMPARE);
-      request.setAttribute("nTotalFile", fileInfos.size());
-
-      // URL can not determine offset and limit, let javascript in jsp determine and redirect
-      if (request.getParameter("offset") == null && request.getParameter("limit") == null) {
-        getServletContext().getRequestDispatcher(mBrowseJsp).forward(request, response);
-        return;
-      }
-
-      try {
-        int offset = Integer.parseInt(request.getParameter("offset"));
-        int limit = Integer.parseInt(request.getParameter("limit"));
-        List<UIFileInfo> sub = fileInfos.subList(offset, offset + limit);
-        request.setAttribute("fileInfos", sub);
-      } catch (NumberFormatException e) {
-        request.setAttribute("fatalError",
-                "Error: offset or limit parse error, " + e.getLocalizedMessage());
-        getServletContext().getRequestDispatcher(mBrowseJsp).forward(request, response);
-        return;
-      } catch (IndexOutOfBoundsException e) {
-        request.setAttribute("fatalError",
-                "Error: offset or offset + limit is out of bound, " + e.getLocalizedMessage());
-        getServletContext().getRequestDispatcher(mBrowseJsp).forward(request, response);
-        return;
-      } catch (IllegalArgumentException e) {
-        request.setAttribute("fatalError", e.getLocalizedMessage());
-        getServletContext().getRequestDispatcher(mBrowseJsp).forward(request, response);
-        return;
-      }
-
```

Comment:
This request will to forwarded to the browse page, and the browse page know the request from browselog page, so the page display log, otherwise, it will display directories and files in the Alluxio file system.